src/main/java/dev/mlml/Utils.java

```diff
@@ -12,4 +12,44 @@ public static boolean stringIsUserMention(String input) {
     public static boolean stringIsChannelMention(String input) {
         return input.matches("^<#\\d{17,19}>$");
     }
+
+    // This method takes in time in milliseconds and returns a formatted string in H:M:S
+    public static String timePrettyPrint(long milliseconds) {
+        if (milliseconds < 1000) {
+            return "1 second";
```

Comment:
What's the reason for this simplification?

Reply:
You wouldn't want to display 0 seconds remaining. This is just a fast way of returning without additional clutter.